src/platform/datapath_raw_xdp_win.c

```diff
@@ -71,6 +72,14 @@ typedef struct XDP_QUEUE {
     CXPLAT_SQE TxIoSqe;
     XSK_RING TxRing;
     XSK_RING TxCompletionRing;
+    struct {
```

Comment:
Just curious why not flatten this out instead of making it a nested struct? Is this a memory layout optimization?

Reply:
Yeah, this is just copy/pasta. I kept it similar because (a) my own personal preference on the matter doesn't outweigh simply following the existing pattern in this project and (b) it seemed reasonable to me, in terms of being somewhat more compact in memory and more friendly for auto-complete, code searching, etc.